### ISSUE TITLE
one question about kernel 5*5

### ISSUE THREAD

**Issue by alexliyang** [Bisonai/mobilenetv3-tensorflow#5]
when train cifar10, initial size is 96*96 , and when downscale size is less than kernel 5*5 ,how to deal these channels?

**Comment by martinkersner** [comment]
Hi @alexliyang!

Thanks for a good question! The input size 96x96 is just an example. You should use larger input for MobileNetV3 architecture if you want to get correct/valid results. I will update repository to reflect this constraint.

Cheers,
Martin

**Comment by alexliyang** [comment]
and I meet another question,  I want to use this model to classify 7 classes, when training acc is about      
0.9, I save model and test , the reuslt is very bad, always tell me the class id is 2 , no other class ids

**Comment by martinkersner** [comment]
Do you evaluate your model at the end of every epoch? What is the validation accuracy?
